kit/scrolls/ios/UIKitAccordionOverlayView.m

```diff
@@ -232,8 +232,10 @@ - (UIImage *)takeScreenshot:(UIView *)view startY:(NSNumber *)startY endY:(NSNum
     
     CGContextRef context = UIGraphicsGetCurrentContext();
 
-    // TODO
-    [[UIColor whiteColor] setFill];
+    // Not sure if we should copy it though,
+    // did it just not to mess with original `backgroundColor`
+    UIColor *bgColor = [self.backgroundColor copy];
+    [bgColor setFill];
```

Comment:
Since you just do a `setFill` here, there is no need indeed :) But I'm OK with that